ksql-parser/src/main/java/io/confluent/ksql/parser/FormatterUtil.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.parser;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import com.google.common.collect.ImmutableSet;
+
+public class FormatterUtil {
```

Comment:
Do we really need a class for this? It is only used in `SqlFormatter` so can just include it there?

Reply:
I think it will also be used by ExpressionFormatter to correct new bugs similar to this one.

Reply:
For now i'd just include it in `SqlFormatter` - if the same functionality is needed elsewhere we can pull it out later

Reply:
ok